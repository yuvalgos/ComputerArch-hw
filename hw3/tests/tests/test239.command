./dflow_calc tests/test239.dat tests/test239.in p179 d161 p31 p40 p129 d91 d41 d19 d182 p141 d174 d117 d113 p165 d46 d47 p124 p155 p131 d21 d170 d159 d16 p162 p116 p188 p148 p117 d116 p6 p171 d31 p110 p182 d21 d161 p36 p155 d116 d146 p52 d58 d111 p33 p18 p93 d126 d163 d18 d174 p69 p107 p2 d140 p193 d43 d38 d162 d9 p54 d53 d72 d131 d0 p155 p109 d68 d68 d5 d15 p143 p45 p190 d163 d32 p21 d86