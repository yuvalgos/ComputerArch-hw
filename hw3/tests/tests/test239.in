3 10 21 24
5 4 26 31
0 1 3 30
3 19 8 31
5 21 19 18
2 4 26 0
4 27 25 28
1 13 11 28
2 31 27 26
5 13 18 24
6 6 31 30
5 23 9 3
2 31 1 18
7 27 14 6
2 3 27 17
6 2 17 17
3 30 17 0
1 27 16 1
4 5 9 13
4 18 10 21
5 19 8 17
7 30 24 1
1 12 10 18
4 23 7 24
8 27 14 8
0 0 20 12
5 20 23 15
1 20 8 1
8 8 29 17
1 2 14 25
3 6 30 5
6 6 30 24
0 4 1 3
4 29 30 8
7 18 4 20
7 13 22 4
1 27 27 25
7 7 10 17
8 14 6 10
2 25 1 30
1 20 22 7
2 12 16 31
1 21 31 9
5 29 0 6
8 12 10 4
4 22 4 29
0 12 1 28
1 28 19 8
2 5 29 29
3 1 22 13
0 22 27 8
8 26 23 15
7 3 24 2
2 26 16 18
5 4 5 24
7 9 19 9
4 21 12 26
7 14 21 25
6 6 13 4
5 8 2 0
3 12 28 6
1 30 16 2
5 9 10 29
2 17 21 27
5 13 28 21
2 23 27 13
8 12 10 2
3 11 14 16
8 13 13 12
5 23 31 9
7 2 31 19
0 10 22 16
6 29 5 29
0 21 26 13
1 2 3 28
4 21 31 8
0 16 21 1
1 26 24 7
0 18 8 24
5 26 21 8
3 17 22 4
5 5 20 28
1 15 28 3
1 24 19 21
6 27 9 6
7 0 24 14
5 31 1 16
1 19 10 16
5 25 7 20
3 16 21 19
0 25 14 27
6 9 13 23
1 31 7 27
1 14 7 26
8 22 14 20
3 1 29 19
2 13 11 8
8 2 6 0
8 4 11 26
2 18 5 0
1 3 6 15
6 29 31 21
8 3 22 3
3 9 27 26
8 24 15 12
8 22 16 22
5 19 27 28
0 22 29 3
6 11 0 12
7 17 19 17
7 28 20 8
1 11 19 30
4 3 9 6
8 9 26 27
5 1 0 2
7 2 13 14
4 19 18 24
6 6 2 0
6 19 6 13
5 20 13 3
5 25 16 31
5 20 29 17
1 19 29 25
0 22 30 21
8 16 26 0
6 2 26 12
4 3 12 18
2 2 15 13
5 8 29 27
6 29 20 31
7 12 31 0
3 7 19 22
6 16 16 20
7 25 13 20
8 22 27 19
2 3 7 24
3 21 31 3
2 4 2 8
3 21 28 15
5 28 11 7
8 27 23 30
0 10 15 0
7 5 7 13
1 9 5 17
2 17 30 28
3 31 20 8
5 15 1 19
0 27 30 7
4 8 15 10
6 19 4 7
5 11 6 20
2 7 28 23
3 24 8 28
6 22 15 27
7 2 13 18
0 28 27 11
8 28 23 31
0 3 3 26
0 1 22 3
6 30 20 13
6 5 6 29
6 8 23 16
8 25 31 18
7 11 9 13
6 10 10 17
1 21 18 23
8 6 13 25
7 21 18 1
6 16 18 2
5 11 4 14
4 5 12 15
7 17 14 21
6 24 15 10
0 4 5 25
5 14 4 26
0 18 11 22
4 10 31 7
1 6 27 25
5 26 18 0
6 5 15 15
3 13 20 19
7 4 11 13
7 12 7 25
6 25 28 12
7 22 3 5
6 26 19 21
8 23 0 9
5 29 16 16
5 18 8 27
7 29 11 19
2 12 11 19
6 2 13 22
4 7 25 11
3 20 28 6
3 11 5 10
6 15 7 9
4 25 13 8
1 19 11 31
1 20 14 7
8 4 31 22
