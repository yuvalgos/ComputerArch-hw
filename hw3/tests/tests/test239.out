Reading the opcodes latency info from tests/test239.dat ... Got latency for 9 opcodes
Reading the program file tests/test239.in ... Found 200 instructions
getProgDepth()==251
getDepDepth(179)==181
getInstDeps(161)=={69,132}
getDepDepth(31)==21
getDepDepth(40)==26
getDepDepth(129)==157
getInstDeps(91)=={68,69}
getInstDeps(41)=={-1,12}
getInstDeps(19)=={0,4}
getInstDeps(182)=={151,162}
getDepDepth(141)==132
getInstDeps(174)=={173,79}
getInstDeps(117)=={115,85}
getInstDeps(113)=={79,84}
getDepDepth(165)==137
getInstDeps(46)=={2,-1}
getInstDeps(47)=={20,28}
getDepDepth(124)==132
getDepDepth(155)==174
getDepDepth(131)==135
getInstDeps(21)=={-1,2}
getInstDeps(170)=={130,146}
getInstDeps(159)=={121,96}
getInstDeps(16)=={-1,-1}
getDepDepth(162)==164
getDepDepth(116)==137
getDepDepth(188)==191
getDepDepth(148)==181
getDepDepth(117)==144
getInstDeps(116)=={99,104}
getDepDepth(6)==0
getDepDepth(171)==219
getInstDeps(31)=={21,-1}
getDepDepth(110)==49
getDepDepth(182)==187
getInstDeps(21)=={-1,2}
getInstDeps(161)=={69,132}
getDepDepth(36)==23
getDepDepth(155)==174
getInstDeps(116)=={99,104}
getInstDeps(146)=={114,122}
getDepDepth(52)==3
getInstDeps(58)=={35,54}
getInstDeps(111)=={106,61}
getDepDepth(33)==23
getDepDepth(18)==13
getDepDepth(93)==73
getInstDeps(126)=={66,99}
getInstDeps(163)=={143,96}
getInstDeps(18)=={-1,9}
getInstDeps(174)=={173,79}
getDepDepth(69)==64
getDepDepth(107)==132
getDepDepth(2)==0
getInstDeps(140)=={69,61}
getDepDepth(193)==190
getInstDeps(43)=={25,31}
getInstDeps(38)=={31,0}
getInstDeps(162)=={145,99}
getInstDeps(9)=={-1,-1}
getDepDepth(54)==56
getInstDeps(53)=={-1,34}
getInstDeps(72)=={48,43}
getInstDeps(131)=={122,123}
getInstDeps(0)=={-1,-1}
getDepDepth(155)==174
getDepDepth(109)==125
getInstDeps(68)=={64,66}
getInstDeps(68)=={64,66}
getInstDeps(5)=={-1,-1}
getInstDeps(15)=={-1,-1}
getDepDepth(143)==161
getDepDepth(45)==59
getDepDepth(190)==167
getInstDeps(163)=={143,96}
getInstDeps(32)=={2,14}
getDepDepth(21)==2
getInstDeps(86)=={49,76}
